core/dbt/parser/manifest.py

```diff
@@ -331,7 +331,16 @@ def load_all(
         macro_hook: Callable[[Manifest], Any],
     ) -> Manifest:
         with PARSING_STATE:
-            projects = load_all_projects(root_config)
+            projects = root_config.load_dependencies()
+            v1_configs = []
```

Comment:
I'm testing this out locally, and I'm (strangely) only seeing one project or package name in the deprecation warning. My setup:
 - has a v1 `dbt_project.yml` file in my root project
 - has a v1 `dbt_project.yml` file in a dependency package

Any idea what might be going wrong?